WebRoot/messages/ZmMsg.properties

```diff
@@ -2450,9 +2450,7 @@ offlineHeapDump = Do we have your consent to upload heap dump of Zimbra Desktop\
 offlineMoveFolderError = Folder move failed, please try again later
 OfflineServerReachable = Connected to server. You are back online.
 OfflineServerNotReachable = Cannot connect to server. You are now offline.
-offlineSettings = Offline Mode
 offlineSignOutWarning = Signing out will also turn off the offline mode and clear the locally stored data.<br><br>Are you sure you want to continue?
```

Comment:
Don't we need to remove `offlineSignOutWarning` ?

Reply:
https://github.com/Zimbra/zm-web-client/pull/868#issue-2248326894